### DISCUSSION TITLE
is it logging or am I missing something?

### DISCUSSION THREAD

**Discussion by jimbrayrcp** [ninest/aquaui#5] [Q&A] [answered]
Not sure if I am missing something but I keep getting all this extra logging in my run window. 
Is there a way to turn it off?
**Here is an example of what I am seeing:**
    `osascript -e            set answer to display dialog \"	 The Schedule Importer 
 		       is now ACTIVE 
    \" buttons { \"Go\", \"No\" } default button \"Go\" cancel button \"No\"  with icon POSIX file \"applet.png\" 
                return answer`

**Comment by ninest** [comment, marked as answer]
Thank you for asking! I checked and it seems that I left a print statement before publishing. I just published `aquaui` version `0.0.1-2`, which, from my testing, does not print anything.

Do note that you will still get some errors logged, as a result from the command `osascript`.

For example, if you run this code

```py
from aquaui import Dialog, Buttons

Dialog("Do you want to").with_buttons(Buttons(["Go", "No"], default_button="Go", cancel_button="No")).show()
```

and press "No" on the dialog, or use the escape key to dismiss the dialog, you will see a message like

```bash
14:64: execution error: User cancelled. (-128)
```

I may remove these messages in a future version though
